src/app/views/login/SignInController.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { Component } from "react";
```

Comment:
As per the description, this is a duplicate file that will be edited in future PRs
- line 6 is however different